parts/header.html

```diff
@@ -1,7 +1,7 @@
 <!-- wp:group {"layout":{"inherit":"true"}} -->
 <div class="wp-block-group">
 	<!-- wp:group {"align":"full","layout":{"type":"flex","justifyContent":"space-between"},"style":{"spacing":{"padding":{"bottom":"var(--wp--preset--spacing--50)","top":"var(--wp--preset--spacing--50)"}}}} -->
-	<div class="wp-block-group alignfull" style="padding-bottom:var(--wp--preset--spacing--50);padding-top:var(--wp--preset--spacing--50)">
+	<div class="wp-block-group alignfull" style="padding-bottom:3.75rem;padding-top:3.75rem;width: 1000px">
```

Comment:
As here we are changing the styles from being this `style="padding-bottom:var(--wp--preset--spacing--50);padding-top:var(--wp--preset--spacing--50)"` to `style="padding-bottom:3.75rem;padding-top:3.75rem;width: 1000px"` in the inner div, should the block markup have the same here? `<!-- wp:group {"align":"full","layout":{"type":"flex","justifyContent":"space-between"},"style":{"spacing":{"padding":{"bottom":"var(--wp--preset--spacing--50)","top":"var(--wp--preset--spacing--50)"}}}} -->`? Not sure if this is the reason, the header template is broken for me. The resolving window points to this line of code.

<img width="1538" alt="image" src="https://user-images.githubusercontent.com/6820724/194156106-ce149bfa-323d-4b4f-88db-27d60c4893d4.png">

Reply:
Hmh it is weird, it is not broken for me. I will add the changes and can you check then @Imran92